locales/en.json

```diff
@@ -2285,7 +2286,10 @@
 				"unfortunatelyTheRequestFailedAgain": "Sorry, the request failed again. To continue using it, please reload the page.",
 				"weCouldNotMakeSureThat": "We could not make sure that this request really came from you. Please click the button below to reload the page.",
 				"weCouldNotMakeSureThatThisRequestReallyCameFromYou": "We could not make sure that this request really came from you. Please click the button below to confirm.",
-				"yourRequestCouldNotBeVerified": "Your request could not be verified."
+				"yourRequestCouldNotBeVerified": "Your request could not be verified.",
+				"yourSessionHasExpired": "Your session has expired",
+				"youAreAlreadyLoginInOtherTab": "It appears that you are already logged into Schulcloud in another tabs.",
```

Comment:
again use instance name instead of schulcloud

Reply:
Ready